debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -308,6 +308,30 @@ public static SnapshotMode parse(String value, String defaultValue) {
                     + "'initial' (the default) to specify the connector should always perform an initial sync when required; "
                     + "'never' to specify the connector should never perform an initial sync ");
 
+    public static final Field CONNECT_TIMEOUT_MS = Field.create("connect.timeout.ms")
```

Comment:
Should we prefix the option names with "mongodb.", e.g. "mongodb.connect.timeout.ms"? Looks like we do that for the properties which are mostly through to the client builder.

Reply:
That's probably a good idea, +1.